packages/math/src/react/hooks/useEquationInput.ts

```diff
@@ -79,9 +78,6 @@ export const useEquationInput = ({
         } else if (isHotkey('escape')(e)) {
           e.preventDefault();
           onDismiss();
-        } else if (isHotkey('meta+z')(e)) {
-          e.preventDefault();
-          editor.undo();
         } else if (isHotkey('meta+y')(e) || isHotkey('meta+shift+z')(e)) {
```

Comment:
seems not work in macos cmd + shift + z

Reply:
hmm, what's the expected behavior for cmd shift z in macos?

Reply:
If you really need to do this, you can use `Hotkeys.isUndo` and `Hotkeys.isRedo` from `packages/core/src/lib/utils/hotkeys.ts`.

Reply:
I was careless yesterday and did not see the redo logic below 😥, i think simply deleting it makes the most sense?

Reply:
The effect looks great. @wststone

Reply:
but I found an another history issue.
history issue is really tricky.

https://github.com/user-attachments/assets/8f3725b5-dcbf-4a29-8e81-f3c170fb2b3e

Reply:
Referring to Notion, I think what we need to do is to avoid opening the popover during an undo action.

Reply:
I will take a look